perfkitbenchmarker/benchmarks/scimark2_benchmark.py

```diff
@@ -0,0 +1,265 @@
+# Copyright 2015 Zi Shen Lim. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs SciMark2.
+
+Original documentation & code: http://math.nist.gov/scimark2/
+
+SciMark2 is a Java (and C) benchmark for scientific and numerical computing.
+It measures several computational kernels and reports a composite score in
+approximate Mflops (Millions of floating point operations per second).
+
+For convenience, we use the code base at https://github.com/zlim/scimark2,
+which contains both the Java and C versions of SciMark2 in a single
+repository.
+"""
+
+import logging
+import re
+
+from perfkitbenchmarker import regex_util
+from perfkitbenchmarker import sample
+from perfkitbenchmarker import vm_util
+
+SCIMARK2_URL = 'https://github.com/zlim/scimark2/archive/master.zip'
```

Comment:
My preference on this would be to pull the code directly from NIST and then apply changes in the benchmark code.  This makes it clear the benchmark has not been modified in any way and if changes are made they are transparent.